META.json

```diff
@@ -2,21 +2,21 @@
    "name": "citus",
    "abstract": "Scalable PostgreSQL for real-time workloads",
    "description": "Citus horizontally scales PostgreSQL across commodity servers using sharding and replication. Its query engine parallelizes incoming SQL queries across these servers to enable real-time responses on large datasets.",
-   "version": "10.2.4",
+   "version": "11.0.0-beta",
    "maintainer": "\"Citus Data\" <packaging@citusdata.com>",
    "license": "agpl_3",
    "provides": {
       "citus": {
          "abstract": "Citus Distributed Database",
          "file": "citus.so",
          "docfile": "README.md",
-         "version": "10.2.4"
+         "version": "11.0.0-beta"
```

Comment:
pgxn standard does not accept _ so I need to change the '_' into -
Since we are using pkgvars pkglatest parameter to get the code from github, it will not affect us